backup.py

```diff
@@ -50,19 +53,43 @@ def run_rsync(settings, path):
         raise Exception("rsync returned error code {}".format(p.returncode))
 
 
-def run():
+def run_rsync(settings, path):
+    starport = settings.starport
+    cmd = rsync_cmd(settings.ssh_key_path, path, starport)
+    run_cmd_with_logging(cmd)
+
+
+def run_rsync_from_container(settings, source_volume):
+    starport = settings.starport
+    docker_ssh_key_path = "/etc/id_rsa"
+
+    cmd = ["docker", "run", "--rm", "-i", "-t",
```

Comment:
Is there any reason not to use the python docker module?

Reply:
I don't know, is there an example of its usage in Montagu somewhere so I can compare?

Reply:
service.py in montagu uses it quite a bit (https://github.com/vimc/montagu/blob/master/src/service.py)

The docs for the module are unusually readable https://docker-py.readthedocs.io/en/stable/ and I've found the api nicer than shelling out